src/store.js

```diff
@@ -0,0 +1,15 @@
+import {
+  createStore,
+  applyMiddleware,
+  compose
+} from 'redux';
+import reducer from './reducers';
+import { promiseMiddleware } from 'promise-middleware-redux';
+
+export default createStore(
+  reducer,
+  compose(
+    applyMiddleware(promiseMiddleware),
```

Comment:
compose only takes a function. please move the window.REDUX... down one line in order to render the page.